source/pervasive/bytes.rs

```diff
@@ -215,17 +217,16 @@ pub closed spec fn spec_u64_from_le_bytes(s: Seq<u8>) -> u64
 pub proof fn lemma_auto_spec_u64_to_from_le_bytes()
   ensures
     forall |x: u64|
-      #![trigger spec_u64_to_le_bytes(x)]
     {
-      &&& spec_u64_to_le_bytes(x).len() == 8
+      &&& (#[trigger] spec_u64_to_le_bytes(x)).len() == 8
```

Comment:
Sorry for not asking earlier: why do you need these additional `#[trigger]` annotations? It doesn't look like these are affected by the exec functions being excluded if the flag is absent.

Reply:
I'm not sure, but without changing the `#[trigger]` in the assertion below (the change on this line is cosmetic) somehow the post conditions cannot be proven. Perhaps it's because of some `broadcast_forall` axioms that aren't enabled when `alloc` is disabled.

Reply:
It seems the issue is instability (due to the change in other proofs in the same module), not the triggers. Marking the failing proofs `#[verifier::broadcast_forall]` seems to stabilize them.